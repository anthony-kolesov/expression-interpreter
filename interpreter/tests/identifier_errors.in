# Reference to unknown variable
out undefined1
var me = undefined2

# 'me" should still be undefined
out me

# Check error for case where identifier is declared sometime later.
out g1
var g1 = 2
out g1
print "\n"

# Reference to unknown variable
out 4 + xyz + 4
