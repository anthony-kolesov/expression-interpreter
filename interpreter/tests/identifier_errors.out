ERROR:2:Unknown identifier: undefined1
ERROR:3:Unknown identifier: undefined2
ERROR:6:Unknown identifier: me
ERROR:9:Unknown identifier: g1
2
ERROR:15:Unknown identifier: xyz
